__tests__/data/appsflyer_source_output.json

```diff
@@ -0,0 +1,141 @@
+[
+  {
+    "context": {
+      "library": {
+        "name": "unknown",
+        "version": "unknown"
+      },
+      "integration": {
+        "name": "Appsflyer"
+      }
+    },
+    "integrations": {
+      "Appsflyer": false
+    },
+    "properties": {
+      "idfv": "868049A3-4B2F-4A11-9B00-CFFC362XXXXX",
+      "device_category": "phone",
+      "af_sub1": null,
+      "customer_user_id": "hi.from@appsflyer.example.com",
+      "is_lat": null,
+      "contributor_2_af_prt": null,
+      "bundle_id": "com.appsflyer.AppsFlyer",
```

Comment:
Expected: `message.context.app.namespace`

Reply:
have put in mapping json.